guides/source/routing.md

```diff
@@ -1112,10 +1112,18 @@ For example, here's a small section of the `rake routes` output for a RESTful ro
 edit_user GET    /users/:id/edit(.:format) users#edit
 ```
 
-You may restrict the listing to the routes that map to a particular controller setting the `CONTROLLER` environment variable:
-
+You may restrict the listing to the routes that map to a particular controller using `-c` option or setting the `CONTROLLER` environment variable:
```

Comment:
@davydovanton please remove the mention of `CONTROLLER` here, and in the example below. If you find it anywhere else please :zap: it too :smile: